catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityFilterPlugin.java

```diff
@@ -44,6 +44,12 @@
     }
 
     public void setAttributeMap(List<String> attributeMappings) {
+        if (attributeMappings == null || attributeMappings.size() == 0 || (
+                attributeMappings.size() == 1 && (attributeMappings.get(0)
```

Comment:
Thanks, I think originally I had an `or` here (i think i was checking for null or empty string) and forgot to change it